ibis/pyspark/client.py

```diff
@@ -21,7 +23,18 @@ def execute(self, expr, params=None, limit='default', **kwargs):
 
         if isinstance(expr, types.TableExpr):
             return self.compile(expr).toPandas()
+        elif isinstance(expr, types.ColumnExpr):
+            # expression must be named for the projection
+            expr = expr.name('tmp')
+            return self.compile(expr.to_projection()).toPandas()['tmp']
```

Comment:
What does `to_projection()` do?

Reply:
Turns the column expression into a table projection so self.compile() returns a data frame.